css/index.css

```diff
@@ -150,12 +162,108 @@ footer p {
 }
 
 
-@media all and (max-width: 1200px) {
+
+
+/* style navigational header */
+header ul {
+
+    margin: 100px;
+    font-family: Verdana;
+    font-weight: bold;
+    display: inline;
+    background-color: rgb(136, 6, 136);
+    border: 5px solid black;
+    box-shadow: 0 25px 25px -25px black;
+}
+
+/* made header text next to eachother with space with padding*/
+header ul li {
+
+    display: inline-block;
+    cursor: auto;
+    padding: 13px 35px;
+    list-style: none;
+    position: relative;
+    z-index: 5;
+    transition: 0.3s;
+}
+
+/*hover nav bar*/
+header ul li:hover {
+
+    background: plum;
+}
+
+/* made header with no underline & change link text color*/
+header li a {
+
+    text-transform: uppercase;
+    text-decoration: none;
+    color: black
+}
+
+
+/* made header sticky*/
+header {
+
+    overflow: hidden;
+    position: sticky;
+    top: 0;
+    left: 0;
+}
+
+/*make scrolling smooth*/
+html {
+    scroll-behavior: smooth;
+}
+
+/*added flex with columns*/
+section h2,
+h3,
+ul,
+li,
+form {
+
+
+    flex-direction: column;
+
+
+    display: flex;
+
+}
+
+/*media query for all desktops*/
+
+@media (max-width: 1200px) {
 
     section,
     body {
+        flex-wrap: wrap;
         background-color: teal;
         border: 20px solid grey;
 
     }
+}
+
+/* media query for mobile Phones */
+@media only screen and(max-width : 480px) {
+
+    section,
+    body {
+
+        background-color: skyblue;
+        border: 20px grey;
+
+    }
+}
+
+/* media query for  Tablets */
+@media only screen and(max-device-width:768px) {
```

Comment:
There should be a space after `and` and there should be a space **after** the colon symbol `:`. It would be better to use `max-width` instead of `max-device-width` to stay consistent with the other media queries in this file. The main difference is that `max-width` takes into account whether the device (eg phone/tablet) is in landscape mode or portrait mode, but `max-device-width` does not.

```suggestion
@media only screen and (max-device-width: 768px) {
```

As well, you need to order the media queries from biggest width to smallest width. Otherwise, the way you have it right now,  if you view the webpage on a mobile phone (let's say the phone has a screen width of 300px), the computer will read and apply the CSS for max-width: 480px; then it'll read the media query for 768px, and overwrite any styles it had before. Remember that CSS always reads from top to bottom. 

So your file should have:

```css
/*media query for all desktops*/

@media (max-width: 1200px) {
 /* all styles that should apply only to desktops that are up to 1200px in screen width */
}

/* media query for  Tablets */
@media only screen and (max-width: 768px) {
  /* all styles we want to apply/override for tablets with width up to 768px */
}

/* media query for mobile Phones */
@media only screen and (max-width: 480px) {
  /* all styles we want to apply/override for phones with width up to 480px */
}
```

Reply:
i made changes in my css. i added the header ul to the media query to try to adjust the fonts for the nav bar. i wanted to ask you if you can double look at my css?  I just wanted to make sure my nav bar is looking okay before i merge.